pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter Flows made easy! A Flutter package which simplifies flows w
 repository: https://github.com/felangel/flow_builder
 homepage: https://github.com/felangel/flow_builder
 
-version: 0.0.4
+version: 0.0.5
 
```

Comment:
Why are we bumping the version? We should wait until new features have been merged before bumping the version and publishing to pub.dev

Reply:
[This refactor](https://github.com/felangel/flow_builder/commit/33ac8afab3783c243bcb4d47846222efa9afab35) is already done.

Reply:
In that case, can you also update the CHANGELOG.md?

Reply:
Sure!

Reply:
Done ✅